packages/components/src/Actions/ActionIconToggle/ActionIconToggle.component.js

```diff
@@ -25,7 +25,7 @@ function ActionIconToggle(props) {
 				className={cn}
 				aria-label={label}
 				aria-pressed={active}
-				bsStyle=""
+				bsStyle="default"
```

Comment:
You sure it didn't break the style ? I tested that too but, the colors change a bit (different grey or blue) :s

Reply:
i have use link bsStyle to have the same behaviour now :)